packages/salesforcedx-vscode-replay-debugger/test/unit/checkpoints/checkpointService.test.ts

```diff
@@ -21,6 +22,18 @@ import {
 import { MAX_ALLOWED_CHECKPOINTS } from '../../../src/constants';
 
 describe('Checkpoint Service - unit', () => {
+  const config = vscode.workspace.getConfiguration();
+  const checkpointsEnabled = config.get(
+    'salesforcedx-vscode-replay-debugger-checkpoints.enabled',
+    false /* value to return if one is not defined */
```

Comment:
This doesn't need a comment. Hovering over `config.get()` already says the second param is the default value.